setup.py

```diff
@@ -1,12 +1,9 @@
 from setuptools import setup, find_packages
 
-# dependencies
-deps = ['PyFxA==0.0.6']
-
 setup(name='fxapom',
-      version='1.3',
+      version='1.3.1',
       description="Mozilla Firefox Accounts Page Object Model",
-      long_description=open('README.md').read(),
+      long_description=open('README.rst').read(),
       classifiers=[],  # Get strings from http://pypi.python.org/pypi?%3Aaction=list_classifiers
```

Comment:
We should add classifiers :smile: at least 'License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)'